src/dotnet-install.ps1

```diff
@@ -195,7 +195,7 @@ function Get-CLIArchitecture-From-Architecture([string]$Architecture) {
         { $_ -eq "x86" } { return "x86" }
         { $_ -eq "arm" } { return "arm" }
         { $_ -eq "arm64" } { return "arm64" }
-        default { throw "Architecture not supported. If you think this is a bug, report it at https://github.com/dotnet/sdk/issues" }
+        default { throw "Architecture not supported. If you think this is a bug, report it at https://github.com/dotnet/install-scripts/issues" }
```

Comment:
would it add value to include the specified architecture which is not supported?

@AR-May  I know, not your change...while we're here, would the above make our exception better?

@bozturkMSFT , what do you think?

Reply:
For me, it sounds reasonable. I can edit the message like that: "Architecture '$Architecture' not supported.".
@bozturkMSFT, what do you think of this?

Reply:
Not a "must", in my opinion, but it would be nice to have.